evm/src/cpu/kernel/asm/core/call.asm

```diff
@@ -18,7 +18,6 @@ global sys_call:
     %address %set_new_ctx_caller
     DUP5 %set_new_ctx_value
     DUP5 DUP5 %address %transfer_eth
-    %set_new_ctx_parent_ctx
```

Comment:
Here and below, `%set_new_ctx_parent_ctx` is already called in `%create_context`.